changelogs/fragments/76681-ansible-galaxy-add-gpg-signature-verification.yaml

```
@@ -0,0 +1,23 @@
+minor_changes:
+- >-
+  ``ansible-galaxy collection [install|verify]`` - use gpg to verify the authenticity of
+  the signed MANIFEST.json with ASCII armored detached signatures provided by the Galaxy
+  server. The keyring (which is not managed by ``ansible-galaxy``) must be provided with
+  the "--keyring" option to use signature verification.
+  If no "--keyring" is specified and the collection to install|verify has associated
```

Consider consistently marking the CLI/cmd bits as inline code rather than just double-quote them.
```suggestion
  the ``--keyring`` option to use signature verification.
  If no ``--keyring`` is specified and the collection to ``install|verify`` has associated
```